tests/UI/pages/BO/BObasePage.js

```diff
@@ -26,6 +26,14 @@ class BOBasePage extends CommonPage {
     this.userProfileLogoutLink = 'a#header_logout';
     this.shopVersionBloc = '#shop_version';
     this.headerShopNameLink = '#header_shopname';
+    this.headerQuickDiv = '#header_quick';
+    this.quickAccessDropdownToggle = '#quick_select';
+    this.quickAccessContainer = '#quick-access-container';
+    this.quickAccessLink = idLink => `${this.headerQuickDiv} li:nth-child(${idLink}) a,
+    ${this.quickAccessContainer} a:nth-child(${idLink})`;
```

Comment:
You can add a class named `quick-row-link` to li (on legacy) and a (on migrated pages).
And use it like this : 

```suggestion
    this.quickAccessLink = idLink => `.quick-row-link:nth-child(${idLink}) `;
```